build-image/first-boot.bash

```diff
@@ -132,6 +132,7 @@ echo "OK"
 
 echo -n "$(timestamp) [openHABian] Updating repositories and upgrading installed packages... "
 apt update &>/dev/null
+/usr/bin/apt --yes --fix-broken install &>/dev/null
 apt --yes upgrade &>/dev/null
```

Comment:
Maybe this is just me but I believe that differences like the superfluous `/usr/bin/` make this code worse (error handling, readability, maintainability) than it needs to be.

Reply:
Yep that's just you :) It _should be_ superfluous but nevertheless it's best practice to use the full path  as $PATH isn't necessarily set or you can have multiple binaries of the same name in your path or even shell builtins such as 'echo'.

Reply:
My point was that we should keep it unified. Your single full path doesn't add any value as long as there are two short version "apt"s before and after it. It just messes up the code ;)

Regarding your argument I actually kind of disagree. I know this is an established best practice but I believe the following: openHABian either controls the system or works closely with it. Guessing where a certain command is located is redundant in that context and doesn't really add any value.
Anyhow I'm not making a strong argument here, nor would I be strictly against changing the way we do things in this project.
I am however against random deviations from the established style... :)
Jm2c, please don't take it the wrong way ;)